### ISSUE TITLE
Try syntax `CUTEstModel("...") do nlp` to allow automatic closing of nlp

### ISSUE THREAD

**Issue by abelsiqueira** [JuliaSmoothOptimizers/CUTEst.jl#275]

**Comment by amontoison** [comment]
Not needed anymore, each CUTEst model is independant and have its own finalizer.

**Comment by tmigot** [comment]
ok, so we no longer need to do finalize(nlp) at the end?
If so, I suggest we create a new issue to update the tutorials.

**Comment by amontoison** [comment]
No, we don't need that since the complete refactor of `CUTEst.jl` and the release 1.0.

**Comment by amontoison** [comment]
If you update the tutorials, please remove `CUTEst.select`.
